cfme/tests/networks/nuage/test_inventory.py

```diff
@@ -0,0 +1,20 @@
+import pytest
+
+from cfme.networks.provider.nuage import NuageProvider
+
+pytestmark = [
+    pytest.mark.provider([NuageProvider], scope='module')
+]
+
+
+def test_tenant_details(setup_provider_modscope, provider, with_nuage_sandbox_modscope):
+    sandbox = with_nuage_sandbox_modscope
+    tenant_name = sandbox['enterprise'].name
+    tenant = provider.collections.cloud_tenants.instantiate(name=tenant_name)
+
+    tenant.validate_stats({
+        'num_cloud_subnets': 2,
```

Comment:
Where are these values coming from? They're too 'magic' of numbers, and need to be pulled from somewhere consistent, if that's the sandbox fixture.

Reply:
We know what sandbox we create with the fixture so those numbers have a right to be hard-coded. But I feel you @mshriver since you don't know the sandbox, it's fuzzy, thanks for pointing it out.

May I suggest we fix this in a followup PR because we'll need to refactor return value of the fixture a bit and adjust other Nuage tests as well?